distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+
+[IMPORTANT]
+====
+Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
+====
+
+=== ${branding} Installation Prerequisites
+
+These are the system/environment requirements to configure _prior_ to an installation.
```

Comment:
This section is empty. Should we move what's in section `2.2.5. Java Requirements` here since Java and the file descriptor limit should be configured before anything else can be done? We could also mention the fact that a specific user and group should be created if needed. Thoughts?